### ISSUE TITLE
Request: Template images?

### ISSUE THREAD

**Issue by ghost** [dmmarti/es-theme-showcase#4]
Is there any chance you could release template background and system images for the main menu? I'd like to support more obscure systems like the Dragon 32 and the Fujitsu FM-7, and it would be nuts to ask you to make images for all of them!

Thanks for an excellent theme!

**Comment by Suppressor72** [comment]
I agree.  I would like the templates as well.  I created a background and system image for my game manuals directory to access from within ES.

Also, please release the background images in 1920x1080.  When they are upscaled within ES at least in my RetroPie install, the text and quality is blurry.  It does a poor job of upscaling.

**Comment by Suppressor72** [comment]
I resolved the upscale issue by opening the images in Microsoft Paint and resizing them to 1920x1080, which gives better results that the upscaling by ES, but still not perfect, so having them released at a higher quality resolution (if you have it), would be great.
